cmake/flash/check_runner_dependencies.cmake

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# The purpose of this script is to ensure that no runners targets contains
+# additional dependencies.
+#
+# If the target contains dependencies, an error will be printed.
```

Comment:
Can you add a comment explaining that we aren't using false because we don't require cmake 3.16?

Reply:
Done, but I did it where the script is called.